manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
+  <ProviderNameSpace>Microsoft.Azure.Extensions</ProviderNameSpace>
```

Comment:
@boumenot I think we can just publish as a new extension under our subs as 2.0. I realized the migration from 1.6 to 2.0 has to be by changing all the templates, documentation and all the search engine results and will take time. We might as well publish it as a new ext.